doc/release-notes/rl-2009.adoc

```diff
@@ -1,8 +1,7 @@
 [[sec-release-20.09]]
-== Release 20.09 (unreleased)
+== Release 20.09
 
-This is the current unstable branch and the information in this
-section is therefore not final.
+The 20.03 release branch became the stable branch in late September, 2020.
```

Comment:
```suggestion
The 20.09 release branch became the stable branch in late September, 2020.
```

Copy-pasted but forgot to update the actual version :P

Reply:
Thanks! Fixed now.